core/src/kmx/kmx_plus.cpp

```diff
@@ -375,11 +376,100 @@ COMP_KMXPLUS_TRAN::valid(KMX_DWORD _kmn_unused(length)) const {
     assert(false);
     return false;
   }
-  // TODO-LDML
-  DebugLog("!! More to do here.");
   return true;
 }
 
+COMP_KMXPLUS_TRAN_Helper::COMP_KMXPLUS_TRAN_Helper()
+    : tran(nullptr), is_valid(false), groups(nullptr), transforms(nullptr), reorders(nullptr) {
+}
+
+bool
+COMP_KMXPLUS_TRAN_Helper::setTran(const COMP_KMXPLUS_TRAN *newTran) {
+  if (newTran != nullptr) {
+    // Only print this if not null.
+    DebugLog("tran helper: validating '%c%c%c%c' newTran=%p", DEBUG_IDENT(newTran->header.ident), newTran);
+  }
+  is_valid = true;
+  if (newTran == nullptr) {
+    DebugLog("tran helper: invalid, newTran=%p", newTran);
+    // null = invalid
+    is_valid = false;
+    // No assert here: just a missing layer
+    return false;
+  }
```

Comment:
```suggestion
  is_valid = true;
  if (newTran == nullptr) {
    DebugLog("tran helper: invalid, newTran=%p", newTran);
    // null = invalid
    is_valid = false;
    // No assert here: just a missing layer
    return false;
  }
  DebugLog("tran helper: validating '%c%c%c%c' newTran=%p", DEBUG_IDENT(newTran->header.ident), newTran);
```

Just reduces the verbosity of the code a little